content/docs/configuration.md

```diff
@@ -256,31 +256,36 @@ The acceptable names are the same as for the package config:
  Key name                     | Type            | Description
 ------------------------------|-----------------|----------------------------------------------------------------------
  `debug`                      | bool            | enable debug logs
- `dry_run`                    | bool            | Do not perform any remote changes (pull requests or comments)
+ `dry_run`                    | bool            | do not perform any remote changes (pull requests or comments)
  `fas_user`                   | string          | username in Fedora account system (to perform kinit if needed)
- `github_token`               | string          | Github API token: this is needed when packit interacts with Github API: https://github.com/settings/tokens (getting full read & write repo scope should be enough)
- `pagure_user_token`          | string          | Pagure token needed to access REST API, get it at: https://src.fedoraproject.org/settings#nav-api-tab
+ `authentication`             | dict            | tokens for services (GitHub, Pagure)
 
-Specifying tokens as environment variables `GITHUB_TOKEN` and
-`PAGURE_USER_TOKEN` is now deprecated and will be removed in future versions.
-Please store the tokens in the config file.
+The `authentication` is a dictionary where:
+* key is a hostname, url or name that can be mapped to a service-type, for example `github.com` or `pagure`
+* value is a dictionary with keys: `token`, `instance_url` (optional) and `type` (optional)
```

Comment:
- `type` -- hard to explain that in few words
  - only if we cannot match the service-type from the service name.
  - if you use something like `github`/`pagure`/`gitlab` in the name, you don't need that

Reply:
OK, I think we can omit it for now since it's not in the example config below.